src/components/Layout/mdx/headers.ts

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from 'react';
```

Comment:
Basically, the whole reason for this is because we can't set `customIds` via `gatsby-remark-autolink-headers`. The plugin offers that functionality, but we're held back by the requirement that this Gatsby project adhere to ES Modules - which is a bit of tech debt @m-hulbert and I have discussed. It holds us back from a number of other `remark` plugins as well.

Small t-shirt, but still a blocker.